add g-release to quick start guide

rajat-soda opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. Please give clear description and fill all the needed fields in the PR template below
2. Provide all the test report and results for the PR. It is mandatory. Otherwise, 
   your PR may get rejected without any review/discussion
3. If the PR is incomplete/in progress, please add [WIP] at the beginning of the PR title.
4. Provide the link to the issue and other relevant files related to the PR
-->

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
 /kind documentation

**What this PR does / why we need it**:
Adds G-release version to the quick start guide. 

**Which issue(s) this PR fixes**:
<!--
*Please provide the issues number or link.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Fixes #106 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED

**Test Report**:
<!--
*Please provide the test report link (public accessible, screen shot or copy paste the test report, 
or add the testing details.
-->
![image](https://user-images.githubusercontent.com/65751939/94394593-55fa3c00-017b-11eb-920a-697b494c75e1.png)
![image](https://user-images.githubusercontent.com/65751939/94514364-9c649f00-023e-11eb-9b5f-ee4cf79d0eaf.png)

**Special notes for your reviewer**:

rajat-soda commented:
@anvithks @kumarashit  Please merge

kumarashit commented:
@rajat-soda This will remove the link for Faroe release. That's not correct as Faroe release notes still will refer to old link and will break. Can we add new section? 
Or a better solution is to put the version number as VARIABLE and keep just one text. And just update the release tag mapping.
Ex:
**********************
|RELEASE NAME | RELEASE TAG|
|Greenland | v1.1.0|
|Faroe | v1.0.0|
|Elba | v0.20.0|

Install SODA Release
Quick installation from SODA Release:

Prerequisite

OS: Ubuntu 16.04 or Ubuntu 18.04

root user rights is REQUIRED
.......
......
.......

Get and Install Release Binaries

wget https://github.com/sodafoundation/installer/releases/download/<RELEASE_TAG>/installer-<RELEASE_TAG>.tar.gz OR Download the installer binaries from
https://github.com/sodafoundation/installer/releases/tag/<RELEASE_TAG>

rajat-soda commented:
> @rajat-soda This will remove the link for Faroe release. That's not correct as Faroe release notes still will refer to old link and will break. Can we add new section?
> Or a better solution is to put the version number as VARIABLE and keep just one text. And just update the release tag mapping.
> Ex:
> 
> |RELEASE NAME | RELEASE TAG|
> |Greenland | v1.1.0|
> |Faroe | v1.0.0|
> |Elba | v0.20.0|
> 
> Install SODA Release
> Quick installation from SODA Release:
> 
> Prerequisite
> 
> OS: Ubuntu 16.04 or Ubuntu 18.04
> 
> root user rights is REQUIRED
> .......
> ......
> .......
> 
> Get and Install Release Binaries
> 
> wget https://github.com/sodafoundation/installer/releases/download/<RELEASE_TAG>/installer-<RELEASE_TAG>.tar.gz OR Download the installer binaries from
> https://github.com/sodafoundation/installer/releases/tag/<RELEASE_TAG>

Fixed and updated the PR

skdwriting commented on content/soda-gettingstarted/quickstart.md:
```diff
@@ -11,9 +11,10 @@ This is a quickstart guide for users and new contributors to try and get familia
-### Install SODA v1.0.0 Faroe Release Jun 2020
 
-Quick installation from SODA Faroe Release:
+### Install SODA v1.1.0 Greenland Release September 2020
```
We can remove this

rajat-soda replied:
Removed

skdwriting commented on content/soda-gettingstarted/quickstart.md:
```diff
@@ -11,9 +11,10 @@ This is a quickstart guide for users and new contributors to try and get familia
 
-Quick installation from SODA Faroe Release:
+### Install SODA v1.1.0 Greenland Release September 2020
+Quick installation from SODA Greenland Release:
```
We can remove this

rajat-soda replied:
Removed

skdwriting commented on content/soda-gettingstarted/quickstart.md:
```diff
@@ -11,9 +11,10 @@ This is a quickstart guide for users and new contributors to try and get familia
+### Install SODA v1.1.0 Greenland Release September 2020
+Quick installation from SODA Greenland Release:
+
 
```
We add a section to mention the releases supported with bulletins. Reverse chronological order
Greenland
Faroe
Elba
Then continue the quick start info\

Please remove the elba from the past release section and update properly

rajat-soda replied:
Removed Elba from Past release section.

kumarashit reviewed (approved):
LGTM

skdwriting reviewed (approved):
LGTM
